docs/src/pages/guides/lazy.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Lazy loading
+description: Learn when to use lazy loading.
+order: 5
+layout: page
+contentType: guide
+products:
+  - Documentation
+prependJs:
+  - "import Lazy from '../../../../src/components/lazy/lazy';"
+---
+
+{{<Lazy lazyClasses="h30-mm h36" lazyComponent={() => import('../../../../src/components/search/examples/basic.js')} />}}
```

Comment:
✏️ Added this as a testing demo. This page will be the future home of lazy loading documentation -- to be completed in a future PR.